draft-barnes-acme.md

```diff
@@ -1093,6 +1093,26 @@ type (required, string):
 To validate a DNS challenge, the server queries for TXT records under the validation domain name.  If it receives a record whose contents match the token in the challenge, then the validation succeeds.  Otherwise, the validation fails.
 
 
+## Wildcard Domain Validation
+
+When the identifier being validated is a domain name, it may contain a wildcard character "\*" as its left-most label (e.g., "\*.foo.example.com"). A wildcard certificate contains a wildcard domain in a CN or subjectAltName of type DNS-ID and automatically vouches for any and all host names within the bare domain name (the FQDN to the immediate right of the wildcard label).
+
+Domain validation for issuance of a wildcard certificate proceeds as follows:
+1. The server MUST validate that the identifier contains a single wildcard character and that the wildcard character is the left-most label. For instance, "foo.\*.example.com" is invalid.
+2. The server MUST validate that the wildcard label does not fall immediately to the left of a registry-controlled or public suffix. For instance, "\*.co.uk" is invalid.
```

Comment:
It's confusing because the CAB baseline requirements say, "If a wildcard would fall within the label immediately to the left of a registry-controlled or public suffix, CAs MUST refuse issuance...". You'd think that if the PSL were a superset, it would just say "public suffix".